lua/satellite/config.lua

```diff
@@ -7,6 +7,7 @@
 ---@field enable boolean
 ---@field overlap boolean
 ---@field priority integer
+---@field signs table<string, string>
```

Comment:
Not super familiar with type hints in Lua - is this right?

Reply:
Yes